_blog/installing-docker-and-kubernetes-on-windows/index.md

```diff
@@ -0,0 +1,477 @@
+
+You can test your minikube installation with:
+
+```bash
+minikube start --vm-driver=hyperv --memory=2048
+```
+
+> Please note that `--vm-driver=hyperv --memory=2048` are necessary only for the first start. If you wish to change the driver or the memory you have to `minikube destroy` and recreate the VM with the new settings.
+
+If for any reason minikube fails to start up, you can debug it with:
+
+```bash
+minikube delete
+minikube start --vm-driver=hyperv --memory=2048 --v=7 --alsologtostderr
+```
+
+The extra verbose logging should help you get to the issue. In my particular case, minikube used to fail with:
+
+> E0427 09:19:10.114873   10012 start.go:159] Error starting host: Error starting stopped host: exit status 1.
+
+Not very helpful. After enabling verbose logging the issue was more obvious:
+
+```text
++ Hyper-V\Start-VM minikube
++ ~~~~~~~~~~~~~~~~~~~~~~~~~
+    + CategoryInfo          : FromStdErr: (:) [Start-VM], VirtualizationException
+    + FullyQualifiedErrorId : OutOfMemory,Microsoft.HyperV.PowerShell.Commands.StartVM
+```
+
+I was running out of memory!
+
+It's time to test if you're installation is successful. In the command prompt type:
+
+```bash
+kubectl get nodes
+```
+
+You should be able to see a single node in Kubernetes.
+
+### Running virtual machines on VirtualBox
+
+Since Hyper-V is now enabled on your device, you won't be able to start any VM on VirtualBox. You can still try and launch your VM. You may experience the following error, though:
+
+> Your PC ran into a problem and needs to restart. We're just collecting some error info, and then we'll restart it for you.
+
+![Blue screen of death]({% link _blog/installing-docker-and-kubernetes-on-windows/bsod.png %})
+
+**If you wish to use VirtualBox or any other executable that uses a Type-2 hypervisor, you should disable it and restart your computer first.**
+
+Open a new command prompt as an administrator and type the following:
+
+```bash
+bcdedit /set hypervisorlaunchtype off
+```
+
+After restarting your device, you should be able to use VirtualBox as usual.
+
+If you wish to use Docker and minikube again, you should enable Hyper-V and restart your machine.
+
+Open a new command prompt as an administrator and type the following:
+
+```bash
+bcdedit /set hypervisorlaunchtype auto
+```
+
+Restart, and now you're ready to run and deploy containers to Kubernetes!
+
+[Jump to the testing your Docker installation](#testing-your-docker-installation)
+
+## 2. Installing Docker and Kubernetes on Windows 10 Home
+
+If you're planning on using Docker and Kubernetes locally, you can probably save yourself from installing other Docker Machine tool such as Docker Toolbox.
+
+You should use minikube as a remote Docker daemon as well as your local Kubernetes cluster.
+
+You can download and install minikube with:
+
+```bash
+choco install minikube -y
```

Comment:
Wow. This is way better than my version. I take back my comments about chocolatey 🥇